### PR TITLE
Full Xcode 16 supports + iOS 18 support

### PR DESCRIPTION
**Pull request by Dimillian**

### CODE REVIEW

**Comment by koyakei** [comment]
Thank you. Im really waiting for it. @Dimillian

File: IceCubesApp/App/Report/ReportView.swift [hunk 1]
```diff
@@ -44,6 +44,7 @@ public struct ReportView: View {
           ToolbarItem(placement: .navigationBarTrailing) {
             Button {
               isSendingReport = true
+              let client = client
```
Comment (TheRogue76) [inline IceCubesApp/App/Report/ReportView.swift:47]: May i ask why this was needed?

Comment (Dimillian) [reply IceCubesApp/App/Report/ReportView.swift:47]: I think it was one of the beta bug, I'll try to remove it.

Comment (Dimillian) [reply IceCubesApp/App/Report/ReportView.swift]: Removed

**Pull request merged**